test/uk/gov/hmrc/gform/sharedmodel/FieldValueFormatValueSpec.scala

```diff
@@ -27,13 +27,14 @@ class FieldValueFormatValueSpec extends Spec {
     val fieldValue = toFieldValue("""{
            "id": "sum",
            "label": "Label",
-           "value": "${amountA + amountB}"
+           "value": "${amountA + amountB}",
+           "format": "text"
           }
       """)
     fieldValue should beJsSuccess(
       FormComponent(
         FormComponentId("sum"),
-        Text(ShortText.default, Add(FormCtx(FormComponentId("amountA")), FormCtx(FormComponentId("amountB")))),
+        Text(BasicText, Add(FormCtx(FormComponentId("amountA")), FormCtx(FormComponentId("amountB")))),
```

Comment:
`BasicText` is weird beast, do we really want to keep it?

 It is never used with `Text` as it is only constructed for multilines. Maybe we should mandate `format` on multiline as well and get rid of `BasicText`?

Note: It looks like `BasicText` is the same as `format: "text(0, 100000)", we should get rid of it.

Reply:
Alex is fine with making it mandatory for TextArea. Will get rid of BasicText then. So for testing, format=text(0, 100000) should be ok then?

Reply:
Removed BasicText and also updated docs at https://confluence.tools.tax.service.gov.uk/display/DF/Gforms+Text+Formats